miasm2/core/utils.py

```diff
@@ -12,6 +12,30 @@
 pck32 = lambda x: struct.pack('I', x)
 pck64 = lambda x: struct.pack('Q', x)
```

Comment:
Should we deprecate `pck64`, `pck32`, etc.? The endianess not being precised, the execution will depend on the host, not the best option....

Reply:
Not for now: this patch is not intended to fix the endiannes everywhere.
Some code may use them elsewhere.